things/redis/streams.go

```diff
@@ -132,6 +156,30 @@ func (es eventStore) CreateChannel(ctx context.Context, token string, channel th
 	return sch, err
 }
 
+func (es eventStore) BulkCreateChannels(ctx context.Context, token string, channels []things.Channel) ([]things.Channel, error) {
```

Comment:
Not sure about these events. Maybe we should break these into multiple events. @drasko any opinion on this?

Reply:
Looking at this again, I'm wondering if it shouldn't be one event for the entire bulk transaction.

Reply:
Not sure. I'm afraid that we'll have an event that is too large to be sent as a single event. Even processing of this event can take a lot of time. I'm not sure what we should do with this. @nmarcetic @drasko any opinions?

Reply:
I'd say that premature optimisation is mother of all evil.

Let's make it functional now, we'll optimize later.

Reply:
I agree with @anovakovic01 Firing event for each device sounds OK to me. It will cause a high load on ES and I/O but this a cost for bulk actions. This is a true state in the system.

Reply:
@nmarcetic @anovakovic01 agreed.

@nwneisen please proceed according to Aleksandar's and Nikola's guidelines.

Reply:
@anovakovic01 @nmarcetic @drasko Just making sure I understood correctly, the current change I have iterates the slice of things/channels and creates a new event for each, so no change is needed.  Or did I miss something with how events are working?

Reply:
You're right, that's what we want, an event per thing. Marking this one as resolved.